### PR TITLE
Implement order management for M/X/XL

### PR DESCRIPTION
**Pull request by lilyfromseattle**
[PURCHASE-1998]

Update Order Management page for larger screen sizes. Also includes some tweaks to the menu and small adjustments to the page for smaller screens. Shoutout to @xtina-starr for helping me troubleshoot some tricky type errors!

__Before:__
Screen Shot 2020-09-22 at 3 51 41 PM

__After:__
Screen Shot 2020-09-22 at 12 24 04 PM
Screen Shot 2020-09-22 at 12 24 13 PM

[PURCHASE-1998]: https://artsyproduct.atlassian.net/browse/PURCHASE-1998

### CODE REVIEW

File: src/v2/Apps/Purchase/Components/PurchaseHistory.tsx [hunk 1]
```diff
@@ -26,85 +33,236 @@ const StyledImage = styled(Image)`
...
+    case "canceled":
+      return "red100"
+    case "refunded":
+      return "red100"
+    case "confirmed":
+      return "black100"
+    case "fulfilled":
+      return "black100"
+  }
+}
```
Comment (damassi) [inline src/v2/Apps/Purchase/Components/PurchaseHistory.tsx:68]: This function / switch pattern is really nice 👍

File: src/v2/Apps/Purchase/Components/PurchaseHistory.tsx [hunk 2]
```diff
@@ -26,85 +33,236 @@ const StyledImage = styled(Image)`
...
+  const { creditCard, requestedFulfillment } = order
+
   const orderIsInactive =
-    order.state === "ABANDONED" || order.state === "CANCELED"
+    order.state === "CANCELED" || order.state === "REFUNDED"
 
+  const partnerImageUrl = get(partner, p => p.profile.icon.url)
+  const partnerInitials = get(partner, p => p.initials)
+  const creditCardNumber = get(creditCard, c => c.lastDigits)
+  const fulfillment = get(requestedFulfillment, r => r.__typename)
```
Comment (damassi) [inline src/v2/Apps/Purchase/Components/PurchaseHistory.tsx:88]: We no longer need `get` now that we support [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)

**Comment by dzucconi** [comment]
If you rebase with master now `type-check` should pass fine

File: src/v2/Apps/Purchase/Components/PurchaseHistory.tsx [hunk 3]
```diff
@@ -26,85 +33,236 @@ const StyledImage = styled(Image)`
...
+              alignItems="center"
+              justifyContent="center"
+              height="50px"
+              width="50px"
+              mr={1.5}
             >
-              <Text variant="text" letterSpacing="tight">
-                {artwork.artist_names}
+              <StyledImage
+                src={get(artwork, a => a.image.resized.url)}
```
Comment (xtina-starr) [inline src/v2/Apps/Purchase/Components/PurchaseHistory.tsx:108]: Another case here where you can use the optional chaining that Chris mentioned.

**Comment by xtina-starr** [comment]
This order management work is looking good @lilyfromseattle!!

**Comment by zephraph** [comment]
Great work @lilyfromseattle! Let's ship it. 🚀

**Pull request merged**